cmd/restore-index-snapshot/main.go

```diff
@@ -47,12 +48,18 @@ func run() int {
 	var (
 		flagCompression string
 		flagEncoding    string
-		flagIndex       string
+
+		flagIndex          string
+		flagIndex2         string
+		flagBlockCacheSize int64
 	)
 
 	pflag.StringVarP(&flagCompression, "compression", "c", compressionZstd, "compression algorithm (\"none\", \"zstd\" or \"gzip\")")
 	pflag.StringVarP(&flagEncoding, "encoding", "e", encodingNone, "output encoding (\"none\", \"hex\" or \"base64\")")
-	pflag.StringVarP(&flagIndex, "index", "i", "index", "database directory for state index")
+
+	pflag.StringVarP(&flagIndex, "index", "i", "index", "path to database directory for state index")
+	pflag.StringVarP(&flagIndex2, "index2", "I", "index2", "path to the pebble-based index database directory")
```

Comment:
```suggestion
	pflag.StringVarP(&flagIndex2, "index2", "I", "index2", "path to the pebble-based index database directory for storing payloads")
```

Reply:
The long term plan is to move all existing methods from `Library` to `Library2` and then rename

Reply:
(added comments around relevant code pieces about that)